src/store/page-load/index.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type PageloadState = {
+  loadPage: boolean;
+};
+
+export const initialState = {
+  loadPage: false,
```

Comment:
I might have gone with something more boolean-ish in name. `isLoaded`, or `isComplete`?  Then it'd later read as `pageLoad.isComplete`. That might be nice.

I'm not too worried about it since it's super easy to change later.

Reply:
updated, thanks